images/opentelemetry-collector-contrib/configs/latest.apko.yaml

```diff
@@ -2,6 +2,8 @@ contents:
   packages:
     - opentelemetry-collector-contrib
     - opentelemetry-collector-contrib-compat
+    # Required for hostmetrics receivers and certain resourcedetection processors (like eks)
+    - busybox
```

Comment:
this adds a shell which we don't really need. we really just need a few utils here (I think just `uname`)? but we don't individually package `uname` so er on the side of caution for now.